client/extensions/woocommerce/app/settings/shipping/shipping-zone-list-entry.js

```diff
@@ -44,17 +44,27 @@ const ShippingZoneEntry = ( { translate, id, name, methods, currency, loaded, is
 		);
 	}
 
-	const renderMethod = ( methodKey ) => {
-		const method = methods[ methodKey ];
-
+	const renderMethodCell = ( title, summary = '', key = 0 ) => {
 		return (
-			<div key={ methodKey } className="shipping__zones-row-method">
-				<p className="shipping__zones-row-method-name">{ method.title }</p>
-				<p className="shipping__zones-row-method-description">{ getMethodSummary( method, currency ) }</p>
+			<div key={ key } className="shipping__zones-row-method">
+				<p className="shipping__zones-row-method-name">{ title }</p>
+				<p className="shipping__zones-row-method-description">{ summary }</p>
 			</div>
 		);
 	};
 
+	const renderMethod = ( methodKey ) => {
+		const method = methods[ methodKey ];
+		let summary = getMethodSummary( method, currency );
+		if ( ! method.enabled ) {
+			summary = translate( '%(summary)s - Disabled', {
```

Comment:
:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?